daemon/daemon.go

```diff
@@ -397,9 +397,8 @@ func (d *Daemon) HandleRestart(t restart.RestartType, rebootInfo *boot.RebootInf
 			logger.Noticef("%s", err)
 		}
 	}
-	d.rebootInfo = boot.RebootInfo{}
```

Comment:
Super nitpick - this could just be `d.rebootInfo = rebootInfo` no? I means, we no longer care if what is passed in is nil or not.

Reply:
good point, added